### ISSUE TITLE
Error running on iPhone5 with default gdx-setup output

### ISSUE THREAD

**Issue by julienvillegas** [libgdx/libgdx#4470]
#### Issue details
using gdx-setup, out of the box, the iOS sub-project fails to run with default iPhone (iPhone5).
       appName = "my-gdx-game"
        gdxVersion = '1.9.4'
        roboVMVersion = '2.2.0'
        box2DLightsVersion = '1.4'
        ashleyVersion = '1.7.0'
        aiVersion = '1.8.0'

#### Reproduction steps/code
* Download latest version of libgdx-setup
* Create project with default parameters
* install mobidevelop plugin 2.0.0
* import project in Android Studio 2.2.2
* Create iOS configuration, leave default device Type (iPhone 5 / x86_64)
Run project: fails with error 

> Failed to launch application. Error Domain=FBSOpenApplicationServiceErrorDomain Code=1 "The request to open "com." failed." UserInfo={NSLocalizedDescription=The request to open "com.xxxx.game" failed., BSErrorCodeDescription=RequestDenied, NSUnderlyingError=0x7fb0cd80c660 {Error Domain=FBSOpenApplicationErrorDomain Code=1 "(null)" UserInfo={BSErrorCodeDescription=Unspecified}}, NSLocalizedFailureReason=The request was denied by service delegate (SBMainWorkspace) for reason: Unspecified.}

reopen iOS configuration, change device type to iPhone 6, runs. it works.

#### Version of LibGDX and/or relevant dependencies
        gdxVersion = '1.9.4'
        roboVMVersion = '2.2.0'
        box2DLightsVersion = '1.4'
        ashleyVersion = '1.7.0'
        aiVersion = '1.8.0'

#### Please select the affected platforms
- [ ] Android
- [x] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [ ] MacOS

**Comment by intrigus** [comment]
You have to use the Snapshot version of Robovm e.g. 2.2.1-SNAPSHOT

**Comment by intrigus** [comment]
Also: Please edit wiki pages in one revision and not 30 :)

**Comment by julienvillegas** [comment]
Thanks, 
As a person who is not very good technically I struggled a bit with the installation instructions in the wiki. I tried to make it a little more newbie friendly to make sure someone who wants to give LibGDX a go does not get rebuked on the initial steps. Sorry for the multiple edits, I was following the steps one by one.

In regards to this particular issue of RoboVM:
As it stands, a new user, unfamiliar with LibGDX would not be able to run for iPhone 5 with the default install.
Therefore the fix might be to change gdx-setup.jar so that by default it starts with `2.2.1-SNAPSHOT` instead of `2.2.0`.
The aim being: make it as easy as possible for someone to get started.
I think I can make that change (that seems simple enough) if you are in agreement.
